test.html

```diff
@@ -195,8 +200,18 @@ <h2>Test drive</h2>
 	highlightCode();
 }).call(textarea);
 
+// Line Numbers Switcher
+var $lineNumbersBtn = document.getElementById("line-numbers");
+$lineNumbersBtn && $lineNumbersBtn.addEventListener("click", function(){
+
+	document.querySelector("pre").classList.toggle("line-numbers");
```

Comment:
Let's query for this above and reuse the result. Also, let's use `this.checked` to determine whether or not to add / remove the class rather than relying on `toggle` always matching the value.